packages/itwinui-css/backstop/tests/information-panel.html

```diff
@@ -1380,44 +1380,38 @@ <h1>Information panel</h1>
               class="iui-tabs-wrapper iui-horizontal"
               id="demo-pill-2"
             >
-              <ul
+              <div
                 class="iui-tabs iui-pill iui-not-animated"
                 role="tablist"
                 aria-label="Tab group name"
               >
-                <li>
```

Comment:
Is there a semantic loss when not using `<ul>`s and `<li>`s?

Reply:
nope. the list does nothing except get in the way.

we are overriding the `role` anyway, so it doesn't matter what element we use underneath.